File: API/Features/Invoicing/ViewModels/InvoicingPortVM.cs
using System.Collections.Generic;

namespace API.Features.Invoicing {

    public class InvoicingPortVM {

        public string Port { get; set; }
        public IEnumerable<HasTransferGroupVM> HasTransferGroup { get; set; }
        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public int TotalPersons { get; set; }

    }

    public class HasTransferGroupVM {

        public bool HasTransfer { get; set; }
        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public int TotalPersons { get; set; }

    }

}

File: FrontEnd/src/app/features/invoicing/classes/view-models/invoicing-port-vm.ts
export class InvoicingPortVM {

    constructor(

        public port: string,
        public hasTransferGroup: HasTransferGroupVM[],
        public adults: number,
        public kids: number,
        public free: number,
        public totalPersons: number,
        public totalPassengers: number

    ) { }

}

export class HasTransferGroupVM {

    constructor(

        public hasTransfer: boolean,
        public adults: number,
        public kids: number,
        public free: number,
        public totalPersons: number,
        public totalPassengers: number

    ) { }

}